src/WeatherCalendar.cpp

```diff
@@ -0,0 +1,91 @@
+#include <iostream>
+#include "WeatherCalendar.h"
+
+WeatherCalendar::WeatherCalendar(){}
+WeatherCalendar::WeatherCalendar(int len){
+    this->len=len;
```

Comment:
change name variable, not  clear

Reply:
len of array 
Acronym  Definition
len            length